### PR TITLE
added txtpen support in jekyll

### PR DESCRIPTION
**Pull request by rickyhan**

### CODE REVIEW

**Comment by pnn** [comment]
could you provide a few more details regarding the background and purpose of your pull request?

File: docs/_tutorials/convert-existing-site-to-jekyll.md [hunk 1]
```diff
@@ -484,6 +484,7 @@ You can also auto-generate your sitemap by adding a gem called [`jekyll-sitemap`
 For other services you might need (such as contact forms, search, comments, and more), look for third-party services. For example, you might use the following:
 
   * For comments: [Disqus](https://disqus.com/)
+  * For Highlights: [txtpen](https://txtpen/)
```
Comment (pnn) [inline docs/_tutorials/convert-existing-site-to-jekyll.md]: this link just leads nowhere 😞

File: docs/_docs/configuration.md [hunk 2]
```diff
@@ -433,6 +433,7 @@ For example, suppose you set this conditional statement in your code:
 {% raw %}
 {% if jekyll.environment == "production" %}
    {% include disqus.html %}
+   {% include txtpen.html %}
```
Comment (ashmaroli) [inline docs/_docs/configuration.md]: This is not the place to add an include. Also, the file to include in missing in this PR

**Comment by ashmaroli** [comment]
At its current state, this PR feels more like spam :frowning_face:

**Pull request closed without merging**